pkg/rancher-desktop/utils/environment.ts

```diff
@@ -3,4 +3,7 @@
  * @returns True if Rancher Desktop is running in a development or test
  * environment
  */
-export const isDevEnv = /^(?:dev|test)/i.test(process.env.NODE_ENV || '');
+const isDev = /^dev/i.test(process.env.NODE_ENV ?? '');
```

Comment:
Note that `jest` still sets `NODE_ENV` to `test`, so this may or may not be checking what we want. (I wrote up a quick test on this branch to confirm that, at least.)

Reply:
That's correct, I haven't noticed a regression associated with this change, but I think that it's safer to keep checking for `test` as well.